### PR TITLE
main-page

### PR DESCRIPTION
**Pull request by hellokathylee**

### CODE REVIEW

File: Uni_life_tracker/app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -25,7 +25,7 @@
     <TextView
             android:layout_width="377dp"
             android:layout_height="200dp"
-            android:text="@string/uni_life_tracker"
+            android:text="@string/app_name"
```
Comment (EvgeniaSila) [inline Uni_life_tracker/app/src/main/res/layout/activity_main.xml]: Minor detail: but would it be possible to change the font to roboto_mono and roboto_mono_bold where needed

**Review by EvgeniaSila — DISMISSED** [review]
Looks so good!

File: Uni_life_tracker/app/src/main/res/layout/main_nav_drawer.xml [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+<!-- line divider -->
+<!--    <View-->
+<!--            android:layout_width="match_parent"-->
+<!--            android:layout_height="1dp"-->
+<!--            android:layout_marginTop="16dp"-->
+<!--            android:layout_marginBottom="16dp"-->
+<!--            android:background="@color/lavender"-->
+<!--    />-->
+
+    <LinearLayout android:layout_width="match_parent"
```
Comment (dalshekerchi) [inline Uni_life_tracker/app/src/main/res/layout/main_nav_drawer.xml]: You have a couple of styling errors in this file that say "This tag and its children can be replaced by one `<TextView/>` and a compound drawable"

File: Uni_life_tracker/app/src/main/res/layout/main_nav_drawer.xml [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+                android:layout_height="wrap_content"
+                android:text="@string/school"
+                android:fontFamily="@font/roboto_mono_bold"
+                android:textStyle="bold"
+                android:textAlignment="center"
+                android:textColor="@color/black"
+                android:padding="5dp"/>
+    </LinearLayout>
+
+    <LinearLayout android:layout_width="match_parent"
```
Comment (dalshekerchi) [inline Uni_life_tracker/app/src/main/res/layout/main_nav_drawer.xml]: Same here

File: Uni_life_tracker/app/src/main/res/layout/main_nav_drawer.xml [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+                android:layout_height="wrap_content"
+                android:text="@string/life"
+                android:fontFamily="@font/roboto_mono_bold"
+                android:textStyle="bold"
+                android:textAlignment="center"
+                android:textColor="@color/black"
+                android:padding="5dp"/>
+    </LinearLayout>
+
+    <LinearLayout android:layout_width="match_parent"
```
Comment (dalshekerchi) [inline Uni_life_tracker/app/src/main/res/layout/main_nav_drawer.xml]: Here as well

File: Uni_life_tracker/app/src/main/res/layout/main_nav_drawer.xml [hunk 5]
```diff
@@ -0,0 +1,133 @@
...
+                android:layout_height="wrap_content"
+                android:text="@string/expenses"
+                android:fontFamily="@font/roboto_mono_bold"
+                android:textStyle="bold"
+                android:textAlignment="center"
+                android:textColor="@color/black"
+                android:padding="5dp"/>
+    </LinearLayout>
+
+    <LinearLayout android:layout_width="match_parent"
```
Comment (dalshekerchi) [inline Uni_life_tracker/app/src/main/res/layout/main_nav_drawer.xml]: And this

File: Uni_life_tracker/app/src/main/res/layout/main_menu.xml [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+    <ImageView
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:onClick="ClickMenu"
+            android:src="@drawable/ic_menu"
+            android:contentDescription="@string/app_name"
+    />
+
+<!--    <TextView-->
```
Comment (dalshekerchi) [inline Uni_life_tracker/app/src/main/res/layout/main_menu.xml]: Do you need this commented code here?

**Review by dalshekerchi — CHANGES_REQUESTED** [review]

File: Uni_life_tracker/app/src/main/java/com/generic/ult/recyclerAdapter.java [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+import androidx.annotation.NonNull;
+import androidx.recyclerview.widget.RecyclerView;
+
+import java.util.ArrayList;
+
+public class recyclerAdapter extends RecyclerView.Adapter<recyclerAdapter.MyViewHolder>{
```
Comment (dalshekerchi) [inline Uni_life_tracker/app/src/main/java/com/generic/ult/recyclerAdapter.java]: The same of the class should be RecyclerAdapter, make sure to refactor when renaming

File: Uni_life_tracker/app/src/main/res/layout/list_events.xml [hunk 8]
```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (dalshekerchi) [inline Uni_life_tracker/app/src/main/res/layout/list_events.xml]: Do you want to make this activity a LinearLayout like the other ones

**Review by dalshekerchi — CHANGES_REQUESTED** [review]

**Review by dalshekerchi — DISMISSED** [review]

**Review by dalshekerchi — APPROVED** [review]

**Pull request merged**